examples/opensora_pku/opensora/utils/ms_utils.py

```diff
@@ -112,8 +133,8 @@ def init_env(
         ms.set_context(
             mode=mode,
             device_target=device_target,
-            ascend_config={"precision_mode": "allow_fp32_to_fp16"},
             device_id=device_id,
+            ascend_config={"precision_mode": "allow_fp32_to_fp16"},
```

Comment:
why add? no mixed precision control in model construct

Reply:
Removed